register_forte_options.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+
+def register_forte_options(forte_options):
+    register_driver_options(forte_options)
+    register_integral_options(forte_options)
+    register_fci_options(forte_options)
+
+def register_driver_options(forte_options):
+    forte_options.add_str('JOB_TYPE', 'NEWDRIVER',
+                     ['NONE',
+                      'ACI',
+                      'PCI',
+                      'CAS',
+                      'DMRG',
+                      'SR-DSRG',
+                      'SR-DSRG-ACI',
+                      'SR-DSRG-PCI',
+                      'DSRG-MRPT2',
+                      'DSRG-MRPT3',
+                      'MR-DSRG-PT2',
+                      'THREE-DSRG-MRPT2',
+                      'SOMRDSRG',
+                      'MRDSRG',
+                      'MRDSRG_SO',
+                      'CASSCF',
+                      'ACTIVE-DSRGPT2',
+                      'DWMS-DSRGPT2',
+                      'DSRG_MRPT',
+                      'TASKS',
+                      'CC',
+                      'NOJOB'], 'Specify the job type')
+
```

Comment:
Is there a difference between 'NONE' and 'NOJOB'?

Reply:
Removed 'NOJOB'. I think this was a hack.